Lib/test/test_capi/test_float.py

```diff
@@ -183,8 +183,14 @@ def test_pack_unpack_roundtrip(self):
     def test_pack_unpack_roundtrip_for_nans(self):
         pack = _testcapi.float_pack
         unpack = _testcapi.float_unpack
+        arch_32bit = (sys.maxsize == 2147483647)
+        if (not arch_32bit) or (sys.platform == 'win32'):
```

Comment:
```suggestion
        if not arch_32bit:
```
I think win32 now redundant.

Reply:
It seems like tests pass on Windows x86 for all sizes. They only fail for 2 and 4 bytes on Linux x86.